concent_api/core/admin.py

```diff
@@ -16,9 +27,53 @@
 from .models import StoredMessage
 from .models import Subtask
 from .models import SubtaskWithTimingColumnsManager
+from .payments.report import create_report
 
 ACTIVE_STATE_NAMES = [x.name for x in Subtask.ACTIVE_STATES]
 PASSIVE_STATE_NAMES = [x.name for x in Subtask.PASSIVE_STATES]
+DECORATORS = [transaction.non_atomic_requests(using='control')]
+
+
+@method_decorator(DECORATORS, name='dispatch')
```

Comment:
You can use decorator directly here, there is no need for `DECORATORS`.

Reply:
Done